packages/probing-frontend/src/pages/MonitoringDetailPage/MonitoringDetail.page.tsx

```diff
@@ -0,0 +1,85 @@
+import { MonitoringQueries } from '@/api/monitoring/monitoring.hooks'
+import { Link, useParams } from '@/router'
+import { MonitoringEserviceTelemetry } from './components/MonitoringEserviceTelemetry'
+import { MonitoringEserviceProbing } from './components/MonitoringEserviceProbing'
+import { MonitoringEserviceDetail } from './components/MonitoringEserviceDetail'
+import { Box } from '@mui/system'
+import { useTranslation } from 'react-i18next'
+import ArrowBackIcon from '@mui/icons-material/ArrowBack'
+import { Typography } from '@mui/material'
+import { useLoadingOverlay } from '@/stores'
+import { delayedPromise } from '@/utils/common.utils'
+import { PageContainerSkeleton } from '@/layout/PageContainer'
+
+export const MonitoringDetailPage = () => {
+  const { t } = useTranslation('common', {
+    keyPrefix: 'detailsPage',
+  })
+
+  const { showOverlay, hideOverlay } = useLoadingOverlay()
+  const { id: eserviceId } = useParams<'MONITORING_DETAIL'>()
+  const { data: eservicesDetail, isInitialLoading: isInitialLoadingEservice } =
+    MonitoringQueries.useGetEserviceData({
+      eserviceId,
+    })
+
+  const {
+    data: eservicesProbingDetail,
+    isSuccess: isSuccessProbing,
+    refetch,
+    isInitialLoading: isInitialLoadingProbing,
+  } = MonitoringQueries.useGetEserviceProbingData({ eserviceId })
+
+  const hasValidData = isSuccessProbing && eservicesDetail
+
+  const handleRefetch = async () => {
+    showOverlay(t('loading'))
+    // We want show the loading overlay for at least 1 second, to avoid flickering
+    await delayedPromise(refetch(), 1000)
+    hideOverlay()
+  }
+
+  return (
+    <>
+      {isInitialLoadingEservice || isInitialLoadingProbing ? (
+        <>
+          <PageContainerSkeleton />
+        </>
+      ) : (
+        <>
```

Comment:
I would avoid ternary expressions in the JSX if it is possible. 

```typescript
if (isInitialLoadingEservice || isInitialLoadingProbing) return <PageContainerSkeleton />
return(
  ...
)
```

I think what's far more clean and readable